director/lib/constants.py

```diff
@@ -10,6 +10,7 @@ class UrlRoot(enum.Enum):
 
     about = "about"
     accounts = "accounts"
+    organisations = "organisations"
```

Comment:
I wonder if we should use `orgs` here (as github does).

Reply:
Resolved in db63dc47a